docs/user-guide/compute-resources.md

```diff
@@ -269,6 +269,86 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 
 We can see that this container was terminated because `reason:OOM Killed`, where *OOM* stands for Out Of Memory.
 
+## Alpha Features
+
+### Opaque Integer Resources (Alpha Feature)
+
+Opaque integer resources were introduced as an alpha feature in Kubernetes
+v1.5. This feature allows cluster operators to advertise new node-level
+resources that would be otherwise unknown to the system. Users can consume
+these resources in pod specs just like CPU and memory. The scheduler takes care
+of the resource accounting so that no more than the available amount is
+simultaneously allocated to pods. So far only resource accounting is
+implemented (but not node-level isolation.)
+
+Opaque integer resources are resources that begin with the prefix
+`pod.alpha.kubernetes.io/opaque-int-resource-`. The API server
+restricts quantities of these resources to whole numbers. Examples of
+_valid_ quantities are `3`, `3000m` and `3Ki`. Examples of _invalid_
+quantities are `0.5` and `1500m`.
+
+There are two steps required to use opaque integer resources. First, the
+cluster operator must advertise a per-node opaque resource on one or more
+nodes. Second, users must request the opaque resource in pods.
+
+To advertise a new opaque integer resource, the cluster operator should
+submit a `PATCH` HTTP request to the API server to specify the available
+quantity in the `status.capacity` for a node in the cluster. After this
+operation, the node's `status.capacity` will include a new resource. The
+`status.allocatable` field is updated automatically with the new resource
+asychronously by the Kubelet. Note that since the scheduler uses the
+node `status.allocatable` value when evaluating pod fitness, there may
+be a short delay between patching the node capacity with a new resource and the
+first pod that requests the resource to be scheduled on that node.
+
+**Example:**
+
+The HTTP request below advertises 5 "foo" resources on node `k8s-node-1`.
+
+_NOTE: `~1` is the encoding for the character `/` in the patch path._
```

Comment:
OK, will add some more info about it (fact that JSON-patch makes use of JSON-pointer may be fairly niche)